src/components/dashboard/Claim.js

```diff
@@ -221,7 +221,7 @@ const Claim = props => {
   const gdstore = GDStore.useStore()
 
   const { entitlement } = gdstore.get('account')
-  const [dailyUbi, setDailyUbi] = useState((entitlement && entitlement.toNumber()) || 0)
+  const [dailyUbi, setDailyUbi] = useState((entitlement && parseInt(entitlement)) || 0)
```

Comment:
parseInt returns NaN value in the case of invalid input
need to add some helper which checks for result and returns 0 if got NaN
and use it everywhere the BN objects were used